examples/mountain_hydrostatic.py

```diff
@@ -157,34 +152,25 @@ def minimum(f):
 advected_fields.append(("theta", SSPRK3(state, theta0, thetaeqn)))
 
 # Set up linear solver
-if hybridization:
-    params = {'mat_type': 'matfree',
-              'ksp_type': 'preonly',
-              'pc_type': 'python',
-              'pc_python_type': 'firedrake.SCPC',
-              # Velocity mass operator is singular in the hydrostatic case.
-              # So for reconstruction, we eliminate rho into u
-              'pc_sc_eliminate_fields': '1, 0',
-              'condensed_field': {'ksp_type': 'fgmres',
-                                  'ksp_rtol': 1.0e-8,
-                                  'ksp_atol': 1.0e-8,
-                                  'ksp_max_it': 100,
-                                  'pc_type': 'gamg',
-                                  'pc_gamg_sym_graph': True,
-                                  'mg_levels': {'ksp_type': 'gmres',
-                                                'ksp_max_it': 5,
-                                                'pc_type': 'bjacobi',
-                                                'sub_pc_type': 'ilu'}}}
-    linear_solver = HybridizedCompressibleSolver(state, solver_parameters=params,
-                                                 overwrite_solver_parameters=True)
-else:
-    # LU parameters
-    params = {'pc_type': 'lu',
-              'ksp_type': 'preonly',
-              'pc_factor_mat_solver_type': 'mumps',
-              'mat_type': 'aij'}
-    linear_solver = CompressibleSolver(state, solver_parameters=params,
-                                       overwrite_solver_parameters=True)
+params = {'mat_type': 'matfree',
+          'ksp_type': 'preonly',
+          'pc_type': 'python',
+          'pc_python_type': 'firedrake.SCPC',
+          # Velocity mass operator is singular in the hydrostatic case.
+          # So for reconstruction, we eliminate rho into u
+          'pc_sc_eliminate_fields': '1, 0',
+          'condensed_field': {'ksp_type': 'fgmres',
+                              'ksp_rtol': 1.0e-8,
+                              'ksp_atol': 1.0e-8,
+                              'ksp_max_it': 100,
+                              'pc_type': 'gamg',
+                              'pc_gamg_sym_graph': True,
+                              'mg_levels': {'ksp_type': 'gmres',
+                                            'ksp_max_it': 5,
+                                            'pc_type': 'bjacobi',
+                                            'sub_pc_type': 'ilu'}}}
+linear_solver = CompressibleSolver(state, solver_parameters=params,
+                                   overwrite_solver_parameters=True)
```

Comment:
If I understand how `overwrite_solver_parameters` is meant to work, we could actually reduce this list to just:
```
params = {'pc_sc_eliminate_fields': '1, 0'}
```
I think this is the only difference with the default solver parameters?

Reply:
Oh good point. I'll chec that out.

Reply:
We are going to attempt to solve this in a separate pull request